src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,55 @@
+
+.calendar {
+
+  padding: 10px;
+  display: flex;
+  flex-wrap: wrap;
+  width: $week-size;
+  margin: 0 auto;
```

Comment:
BEM doesn't allow us to do so. Better add styling for the parent element to center the calendar inside.